File: Python-Basics/loops/moving.py
width = int(input())
length = int(input())
height = int(input())

apartment_space = width * length * height

box = input()

while box != 'Done':
    apartment_space -= int(box)
    if apartment_space <= 0:
        break
    box = input()

if box == 'Done':
    print(f'{abs(apartment_space)} Cubic meters left')
else:
    print(f'No more free space! You need {abs(apartment_space)} Cubic meters more.')

File: CPP-Basics/Moving.cpp
#include <iostream>
#include <string>


using namespace std;

int main () {

    int l, w, h;
    cin >> l >> w >> h;

    int volume = l * w * h;

    while(volume > 0) {

        string boxes;
        cin >> boxes;


            if(boxes == "Done") {
                break;
            }

     volume -= stoi(boxes);

    }

    if(volume < 0){

    cout << "No more free space! You need " << volume * -1 << " Cubic meters more." << endl;

    } else {

    cout << volume << " Cubic meters left." << endl;

    }

    return 0;
}
